powerdnsadmin/templates/user_profile.html

```diff
@@ -152,5 +160,15 @@ <h3 class="box-title">Edit my profile{%  if session['authentication_type'] != 'L
         };
         applyChanges(postdata, $SCRIPT_ROOT + '/user/profile', false, true);
     });
+
+    // copy otp secret code to clipboard
+    function copy_to_clickboard() {
```

Comment:
Could you rename that function to e.g. `copy_otp_secret_to_clipboard()`, move it to `static/custom/js/custom.js` and delete the duplicate inside `register_otp.html`? 

In `user_profile.html`, `custom.js` is already loaded, and for `register_otp.html`, it would be possible to to add `custom.js` to the `js_login` asset bundle defined in `assets.py`.